capplets/about-me/mate-about-me.c

```diff
@@ -431,6 +465,13 @@ about_me_setup_dialog (void)
 	me->login = g_strdup (g_get_user_name ());
 	me->username = g_strdup (g_get_real_name ());
 
+#if HAVE_ACCOUNTSSERVICE
+	manager = act_user_manager_get_default ();
+	lst = act_user_manager_list_users(manager);
+	g_slist_free_full (lst, g_object_unref);
```

Comment:
You are unref-ing user objects... You should not do that at least according to function documentation! Anyway, at this point it seems that you are not using list so you should just remove these two lines.